requirements.txt

```diff
@@ -4,7 +4,7 @@
 # with package version changes made in requirements-app.txt
 
 cffi==1.14.3
-celery[sqs]==3.1.26.post2 # pyup: <4
+celery[sqs]==5.0.5
```

Comment:
I've read quickly CHANGELOGs for v4 and v5.

Do we need to upgrade setting names?
Do we need to change how we call Celery on the CLI? https://docs.celeryproject.org/en/v5.0.5/whatsnew-5.0.html#step-1-adjust-your-command-line-invocation in `scripts/*.sh`
Anything you saw there that doesn't affect us but you'd like to share?

Reply:
I should also mention celery points out a number of setting name changes which we use which are deprecated and will be pulled in v6. This can be wrapped up in another PR to minimize impacts.

CLI is the same based on how we are using it w containers.